src/lib.rs

```diff
@@ -1357,26 +1318,30 @@ impl Build {
             return client;
         }
 
-        struct JoinOnDrop(Option<thread::JoinHandle<Result<(), Error>>>);
+        struct KillOnDrop(Child);
 
-        impl Drop for JoinOnDrop {
+        impl Drop for KillOnDrop {
             fn drop(&mut self) {
-                if let Some(thread) = self.0.take() {
-                    drop(thread.join());
-                }
+                let child = &mut self.0;
+
+                child.kill().ok();
```

Comment:
Nit, but I'm not a fan of using `.ok()` as an idiom to ignore the return value here. I'd suggest using `let _ = ` and a comment.

(Or, possibly, calling `.expect()` since it's probably worth killing the process if this fails. But that'd be a semantic change from the previous code, so *if* done at all that should be a separate PR.)